lib/record_data.py

```diff
@@ -48,9 +49,8 @@ def record_controls( recordQueue=None ):
                 ## If the audio queue exists - make sure to clear it continuously
                 if( recordQueue != None ):
                     recordQueue.queue.clear()
-            
```

Comment:
@chaosparrot any reason we couldn't put a sleep in this loop so it doesn't spin up my cpu?  Like 50ms?

Reply:
Sounds good!